aruco_pose/src/compat.cpp

```diff
@@ -0,0 +1,65 @@
+/*
+ * Compatibility functions for older versions of dependent packages
+ * Copyright (C) 2019 Copter Express Technologies
+ *
+ * Authors: Oleg Kalachev <okalachev@gmail.com>, Alexey Rogachevskiy <sfalexrog@gmail.com>
+ *
+ * Distributed under MIT License (available at https://opensource.org/licenses/MIT).
+ * The above copyright notice and this permission notice shall be included in all
+ * copies or substantial portions of the Software.
+ */
+
+/*
+ * Code is based on https://github.com/opencv/opencv_contrib, which is distributed
```

Comment:
I'm not sure something in this files is based on that repo.

Reply:
The code in this file is directly lifted from opencv_contrib, see: https://github.com/opencv/opencv_contrib/blob/master/modules/aruco/src/aruco.cpp#L1079-L1109